import sys


def error(msg):
    print(msg)
    sys.exit(1)
